File: ios/Sources/MkMath.swift
/// 数学関連処理
public class MkMath {

    /// 剰余計算
    public static func mod(_ x: Int, _ n: Int) -> Int {
        var candidate = x % n
        if candidate < 0 {
            candidate = candidate + n
        }
        return candidate
    }
}

File: android/src/main/java/com/github/tshion/mktools_android/MkMath.kt
package com.github.tshion.mktools_android

/**
 * 数学関連処理
 */
public object MkMath {

    /**
     * 剰余計算
     *
     * @return 余り(正の範囲)
     */
    public fun mod(x: Int, n: Int): Int {
        var candidate = x % n
        if (candidate < 0) {
            candidate += n
        }
        return candidate
    }
}
